### PR TITLE
Fix normalize() sqrt call to be sure it's using floats

### PR DESCRIPTION
**Pull request by lgritz**
By Alex Wells

### CODE REVIEW

File: src/liboslexec/llvm_ops.cpp [hunk 1]
```diff
@@ -753,14 +753,14 @@ osl_normalize_vv (void *result, void *a)
     using std::sqrt;
```
Comment (fpsunflower) [inline src/liboslexec/llvm_ops.cpp:753]: The `using std::sqrt` line can be removed then?

Comment (lgritz) [reply src/liboslexec/llvm_ops.cpp:753]: Well what do you know? Somehow we both missed this, I guess.
I'm tempted to abandon this. Alex, do you have a reason why we should keep these changes?

**Comment by AlexMWells** [comment]
either way then, std::sqrt has overload for float.  So take your pick sqrtf or std::sqrt.

**Comment by AlexMWells** [comment]
Looking elsewhere in the file sqrtf was used vs. std::sqrt.  The additional benefit of using sqrtf is the possibility of a compiler warning about loss of precision should a double be passed to it vs. std::sqrt which will happily just call the overloaded double version.

**Comment by lgritz** [comment]
OK, I'll just remove the original "using".

**Comment by lgritz** [comment]
The Dual version has OSL_LIKELY to explain that the `len > 0.0f` is the likely path. Should we do that here as well?

**Comment by lgritz** [comment]
I'm torn. `sqrtf` means that we'll never have the problem of inadvertently converting to double. But std::sqrt means that if anybody wants to build OSL using doubles (changing the definition of Vec3), it will work properly without needing to modify this function.

**Comment by lgritz** [comment]
std::sqrt I think is my preference. If it chose the "wrong" sqrtf (double), it will still make a warning when it assigns that back to a float result.

**Comment by AlexMWells** [comment]
If you really want to respect Vec3::value_type then alot more changes are needed.  For this, std::sqrt would be fine, just pointing out that you might want to change all of them one way or the other.

About the OSL_LIKELY sure, it could be used there.

**Pull request merged**